solr/core/src/test/org/apache/solr/cloud/TestPullReplica.java

```diff
@@ -552,7 +557,7 @@ private void waitForNumDocsInAllReplicas(int numDocs, Collection<Replica> replic
     }
   }
 
-  private void waitForDeletion(String collection) throws InterruptedException, KeeperException {
+  static void waitForDeletion(String collection) throws InterruptedException, KeeperException {
```

Comment:
+1 on methods that can be static being static - more efficient when done consistently - also, seems like these checks are a bit general to be in this test class, but unrelated to your patch.

Reply:
that's right, I'm not trying to refactor this test, mainly just trying to use some of the same checks for the `TestPullReplicaWithAuth` test